packages/riipen-ui/src/components/Input.jsx

```diff
@@ -125,14 +132,36 @@ const Input = ({
           border-color: ${theme.palette.negative.main};
         }
 
-        .underlined {
-          background-color: transparent;
-          border: none;
-          border-bottom: 1px solid rgba(0, 0, 0, 0.23);
+        .icon {
+          color: ${theme.palette.grey[500]};
+          left: ${theme.spacing(4)}px;
+          position: absolute;
         }
 
-        .warning {
-          border-color: ${theme.palette.warning.main};
+        .icon.small {
+          font-size: ${theme.typography.body2.fontSize};
+          padding: 7px;
```

Comment:
Also not stoked on the custom padding to get it to align nicely left :/

Reply:
Meh. I think it's fine.

This CSS is assuming the icon size is settable using CSS `font-size`. This might not be true for all iconography libraries. But I am not sure there is a better solution. If people ever use an icon outside of normal svgs they can just put in their own CSS to make the sizing work.